composer.json

```diff
@@ -18,34 +18,27 @@
         }
     ],
     "require": {
-        "php": "^8.0",
-        "ext-openssl": "*",
+        "php": "^8.1",
+        "ext-ffi": "*",
         "ext-json": "*",
         "composer/semver": "^1.4.0|^3.2.0",
-        "amphp/amp": "^2.5.1",
-        "amphp/byte-stream": "^1.8",
-        "amphp/dns": "^1.2.3",
-        "amphp/hpack": "^3.1.0",
-        "amphp/http-server": "^2.1",
-        "amphp/log": "^1.1",
-        "amphp/process": "^1.1.1",
-        "amphp/serialization": "^1.0",
-        "amphp/socket": "^1.1.3",
-        "amphp/sync": "^1.4.0",
-        "amphp/cache": "^1.4.0",
-        "amphp/windows-registry": "v0.3.3",
-        "guzzlehttp/guzzle": "^6.5.8|^7.4.5",
-        "phpunit/phpunit": ">=8.5.23 <10",
-        "tienvx/composer-downloads-plugin": "^1.1.0"
+        "symfony/process": "^5.4|^6.0|^7.0",
+        "guzzlehttp/psr7": "^2.4.5",
+        "tienvx/composer-downloads-plugin": "^1.2.0"
```

Comment:
I know this is a fork of civicrm. I'd feel so much more comfortable if this could be brought under PACT or get a lockfile to ensure the hash of the version is known and audited. This is going to become a runtime package for everyone using PACT going forward.

Reply:
Replaced in https://github.com/pact-foundation/pact-php/pull/544